packages/@webex/internal-plugin-mercury/src/mercury.js

```diff
@@ -196,16 +196,9 @@ const Mercury = WebexPlugin.extend({
         return socket.open(webSocketUrl, options);
       })
       .then(() => {
-        this.webex.internal.metrics.submitClientMetrics('web-ha-mercury', {
-          fields: {
-            success: true,
-          },
-          tags: {
-            namespace: this.namespace,
-            action: 'connected',
-            url: attemptWSUrl,
-          },
-        });
+        this.logger.info(
+          `${this.namespace}: connected to mercury, success, action: connected, url: ${attemptWSUrl}`
```

Comment:
approving the PR, but I'm not sure if we even need these logs here, because I think we already log success failure of websocket connections somewhere else. @maxinteger recently worked on that.

Reply:
Looks good to me.

I just added the `this.namespace` to all logs for clarity.